app/lib/features/home/pages/dashboard.dart

```diff
@@ -91,6 +91,17 @@ class Dashboard extends ConsumerWidget {
               SliverAppBar(
                 backgroundColor: Colors.transparent,
                 actions: <Widget>[
+                  Visibility(
+                    visible: isDesktop,
+                    child: IconButton(
+                      onPressed: () => context.pushNamed(Routes.actionAddUpdate.name),
```

Comment:
So to clarify, this works only on mobile, not desktop? I doubt desktop would support The text slide creation based on editor.

Reply:
Only desktop for now is fine.

Reply:
@gtalha07 
Yes we have support of new update feature in desktop as well. We have different page routers of update view part so need to manage addUpdate button based on the platform.

Reference VIdeo:
https://github.com/acterglobal/a3/assets/51526480/894f349e-e0ce-4ec1-9cf1-7e20fe97d4f6